config/application.rb

```diff
@@ -48,8 +48,5 @@ class Application < Rails::Application
 
     # Use default language as fallback if translation is missing
     config.i18n.fallbacks = true
-
-    # Defer loading of images until it reaches a calculated distance from the viewport
-    config.action_view.image_loading = "lazy"
```

Comment:
Why did you remove this configuration? If I had to guess, this line is what is causing your PR to work in Safari.